.husky/pre-push

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# disable CI
+[ -n "$CI" ] && exit 0
+
+npm run lint
```

Comment:
Do we really want to prevent pushing if linting fails? This will block me from pushing any intermediate work from my local machine. I'm not sure, if I like this approach.

Reply:
Linting on serverside is already implemented, so i guess this should be enough. I've removed it.